.github/actions/ocm-oci-build/action.yaml

```diff
@@ -0,0 +1,119 @@
+name: Build and Push OCI-Image / OCM-Resource
+description: |
+  Builds an OCI Image using `docker/build-push-action@v6`, and exposes a (customisable)
+  OCM Resource fragment as output (for being added to OCM Component-Descriptor).
+  For convenience, the action tries to authenticate against target-OCI-Registry using
+  github.com/gardener/cc-utils/.github/actions/gar-auth.
+inputs:
+  name:
+    description: |
+      resource-name (used in OCM Resource Fragment; must be unique within component)
+    required: true
+  repository:
+    description: |
+      OCI Repository to push to (including registry, but excluding tag)
+      For example: `europe-docker.pkg.dev/gardener-project/snapshots/my-image`
+    required: true
+  version:
+    description: |
+      Used for both image-tag, and OCM Resource fragment.
+    required: true
+  extra-tags:
```

Comment:
The inputs `extra-tags`, `target` and `dockerfile` seem to not being actually used. Also, I think we should allow to specify a custom extra identity for the emitted OCM resource.

Reply:
done - still need to do some testing, though